### PR TITLE
refactor(label): remove unused variable

### PR DESCRIPTION
**Pull request by wsuwt**
## Description
- remove unused variable and add comments to it.

reference to SonarCloud recommendation https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=BUG&id=refinitiv_refinitiv-ui&open=AX-qmzpaqePuut67JdEz

### CODE REVIEW

File: packages/elements/src/label/index.ts [hunk 1]
```diff
@@ -178,8 +178,9 @@ export class Label extends BasicElement {
    * @param [mutation=false] is the request from a mutation event?
    * @returns {void}
    */
+  // mutation is reserved for future used
```
Comment (bualoy-napat) [inline packages/elements/src/label/index.ts]: Why we need to reserve for the future?

File: packages/elements/src/label/index.ts [hunk 2]
```diff
@@ -178,8 +178,9 @@ export class Label extends BasicElement {
    * @param [mutation=false] is the request from a mutation event?
```
Comment (wsuwt) [inline packages/elements/src/label/index.ts]:
```suggestion
   * @param [mutation=false] is the request from a mutation event ? ( reserved for future used )
```

File: packages/elements/src/label/index.ts (hunk 1 above)
Comment (wsuwt) [inline packages/elements/src/label/index.ts]:
```suggestion
```

Comment (bualoy-napat) [reply packages/elements/src/label/index.ts]: I have already updated.

File: packages/elements/src/label/index.ts (hunk 2 above)
Comment (bualoy-napat) [reply packages/elements/src/label/index.ts]: I have already updated.

**Review by bualoy-napat — APPROVED** [review]

**Pull request merged**